apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -127,7 +127,7 @@ export default class AnimationPickerBody extends React.Component {
   };
 
   onCategoryChange = event => {
-    const categoryQuery = event.target.className;
+    const categoryQuery = event.target.className.split(' ')[0];
```

Comment:
This feature relies on the class(es) associated with a target animation to determine if it should be included in the filtered results displayed when changing categories. Because we are now using a SCSS module, all targets have multiple classes. These SCSS classes are always added _after_ the class representing an animation's category.

Reply:
Update: We refactored this to use a data-* attribute instead of a class.